stickynotes/stickynotes_applet_callbacks.c

```diff
@@ -75,10 +75,30 @@ stickynote_show_notes (gboolean visible)
     }
 }
 
+static show_all_stickynote (void)
+{
+    StickyNote *note;
+    GList *l;
+
+    stickynotes->visible = TRUE;
+
+    for (l = stickynotes->notes; l; l = l->next) {
+        note = l->data;
+        stickynote_set_visible (note, TRUE);
+    }
+}
```

Comment:
This function looks pretty similar to **stickynote_show_notes**. Have you tried adding an argument for reusing the existing function?